### PR TITLE
Various small fixes to README.md

### PR DESCRIPTION
**Pull request by johnlunney**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,23 +29,23 @@ The Santa docs are stored in the [Docs](https://github.com/google/santa/blob/mas
...
 
-* Failsafe cert rules: You cannot put in a deny rule that would block the certificate used to sign launchd, a.k.a. pid 1, and therefore all components used in macOS. The binaries in every OS update (and in some cases entire new versions) are therefore auto-whitelisted. This does not affect binaries from Apple's App Store, which use various certs that change regularly for common apps. Likewise, you cannot blacklist Santa itself, and Santa uses a distinct separate cert than other Google apps.
+* Failsafe cert rules: You cannot put in a deny rule that would block the certificate used to sign launchd, a.k.a. pid 1, and therefore all components used in MacOS. The binaries in every OS update (and in some cases entire new versions) are therefore auto-whitelisted. This does not affect binaries from Apple's App Store, which use various certs that change regularly for common apps. Likewise, you cannot blacklist Santa itself, and Santa uses a distinct separate cert than other Google apps.
```
Comment (tburgin) [inline README.md]: actually macOS is correct
https://en.wikipedia.org/wiki/macOS

**Comment by johnlunney** [comment]
I guess I am a bit old-school :) Done.

File: README.md (hunk 1 above)
Comment (johnlunney) [reply README.md]: Done.

**Comment by tburgin** [comment]
Many thanks!

**Pull request merged**